src/js/expanderExpanded.js

```diff
@@ -54,7 +57,7 @@ function mdExpanderExpandedDirective($mdUtil, $animateCss) {
       })
       .start()
       .then(function () {
-        if (isHeightSet !== undefined) {
+        if (isHeightSet === true) {
```

Comment:
I really don't see the point of type checking this value. the value will be a bool as defined above